src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -387,6 +387,10 @@ func chained(rootCA string, persistent bool) (http.RoundTripper, error) {
 			ClientSessionCache: clientSessionCache,
 		},
 	}
+	if persistent {
```

Comment:
Oh, the other piece to understand is that MaxIdleTime here is not an idle timeout for our idletiming connection.  It's telling the Transport how long to hang on to pooled keepalive connections.